studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.utils.ts

```diff
@@ -216,21 +218,21 @@ const unescapeLiteral = (value: string) => {
     value.endsWith('bigint[]') ||
     value.endsWith('smallint[]')
   ) {
-    temp = temp.replaceAll('{', '[')
-    temp = temp.replaceAll('}', ']')
+    temp = temp.replace(/{/g, '[')
```

Comment:
jest throws an error for replaceAll, so changed to replace so that we can run our tests

Reply:
`replace != replaceAll`. Need to use regex. ref https://stackoverflow.com/a/63616567